src/index.tsx

```diff
@@ -11,7 +11,7 @@ import { store }       from './store';
 
 ReactDOM.render(
     <Provider store={store}>
-        <Node key={1} nodeID={1} />
+        <Node key={2} nodeID={2} />
```

Comment:
1 was a magic number for the root node... and it should have been documented... but what is 2?